packages/perspective-viewer/src/less/material.less

```diff
@@ -311,27 +331,51 @@ perspective-viewer #app {
                     background: none !important;
                 }
             }
+        }
 
-            perspective-row:hover {
```

Comment:
Might be worth putting style rules for `perspective-computed-column` in a future commit rather than here.